scrunch/datasets.py

```diff
@@ -952,9 +987,9 @@ def create_single_response(self, categories,
                                  expr=expr,
                                  description=description))
 
-        return Variable(
-            self.resource.variables.create(payload).refresh(),
-            self.resource)
+        new_var = self.resource.variables.create(payload).refresh()
```

Comment:
We can save ourselvs the `.refresh()` here since we are not using the shoji entity for anything we don't need to do a GET for a resource we are not using.

Unless make the Variable class accept an optional keyword argument for the full resource, that way we'd save ourselves a future GET trying to lazy fetch the attributes. 

So either way, remove the `.refresh()` calls on creations or have the Variable take an optional full entity resource to avoid it being lazy when we do have the full information.